packages/scm/src/browser/scm-contribution.ts

```diff
@@ -160,6 +193,35 @@ export class ScmContribution extends AbstractViewContribution<ScmWidget> impleme
             execute: () => this.acceptInput(),
             isEnabled: () => !!this.scmFocus.get() && !!this.acceptInputCommand()
         });
+        commandRegistry.registerCommand(SCM_COMMANDS.GOTO_NEXT_CHANGE, {
+            execute: () => this.dirtyDiffNavigator.gotoNextChange()
```

Comment:
Please see https://github.com/eclipse-theia/theia/pull/13104/commits/d988fd815e23660578d5bb113efe9b8ad3e4cd92. The commit message explains the reason.

Reply:
The problem with commit messages is that nobody sees them ;-). If we're doing something that does not make sense just from the code, we should comment in the code: otherwise the next person might "fix the problem".

Reply:
OK. Addressed in https://github.com/eclipse-theia/theia/pull/13104/commits/bf607eadcf272cfed4192191f2a63115f5a8a480.